QUEST_LV_0200.tsv

```diff
@@ -3368,12 +3368,12 @@ QUEST_LV_0200_20150323_003367	I heard the story of the disciple who succeded rem
 QUEST_LV_0200_20150323_003368	The special bishop is at the Great Cathedral.{nl}He died when he was young, but he still stays at the Great Cathedral because of his special mission.
 QUEST_LV_0200_20150323_003369	I tried my best to hide Maven's secret, but..{nl}Naktis is also staying at the Great Cathedral too long.
 QUEST_LV_0200_20150323_003370	Disciple Goda
-QUEST_LV_0200_20150323_003371	The Great Cathedral has collapsed, but it still keeps its beauty.{nl}Especially, the secrets of Maven, the great bishop, looks amazing everytime we see them.
-QUEST_LV_0200_20150323_003372	Maven, the great bishop, contributed everything builiding this Great Cathedral.{nl}All that was collapsed on Medis Diena.. destiny is too harsh.
-QUEST_LV_0200_20150323_003373	Farewell is short, but the relationship lasts long.{nl}Someday we will meet again in the Goddess.
+QUEST_LV_0200_20150323_003371	$The Great Cathedral has collapsed, but it still keeps its beauty.{nl}Especially, the secrets of Maven, the great bishops, they look amazing every time we see them.
```

Comment:
Especially, the secrets of Maven, the great bishops -> Especially, the secrets of Archbishop Maven